plugins/modules/proxysql_query_rules_fast_routing.py

```diff
@@ -0,0 +1,470 @@
+                cols += 1
+                query_data.append(val)
+                if cols == 1:
+                    query_string += " SET " + col + "= %s,"
+                else:
+                    query_string += " " + col + " = %s,"
+
+        query_string = query_string[:-1]
+        query_string += (
+            "WHERE username = %s "
+            "AND schemaname = %s "
+            "AND flagIN = %s"
+        )
+
+        cursor.execute(query_string, query_data)
+        return True
+
+    def delete_rule_config(self, cursor):
+        query_string = "DELETE FROM mysql_query_rules_fast_routing"
+
+        cols = 0
+        query_data = []
+
+        for col, val in iteritems(self.config_data):
+            if val is not None:
+                cols += 1
+                query_data.append(val)
+                if cols == 1:
+                    query_string += " WHERE " + col + " = %s"
+                else:
+                    query_string += " AND " + col + " = %s"
+
+        if cols > 0:
+            cursor.execute(query_string, query_data)
+        else:
+            cursor.execute(query_string)
+        check_count = cursor.rowcount
+        return True, int(check_count)
+
+    def manage_config(self, cursor, state):
+        if state:
+            if self.save_to_disk:
+                save_config_to_disk(cursor)
+            if self.load_to_runtime:
+                load_config_to_runtime(cursor)
+
+    def create_rule(self, check_mode, result, cursor):
+        if not check_mode:
+            result['changed'] = self.create_rule_config(cursor)
+            result['msg'] = "Added rule to mysql_query_rules_fast_routing"
+            self.manage_config(cursor, result['changed'])
+            result['rules'] = self.get_rule_config(cursor)
+        else:
+            result['changed'] = True
+            result['msg'] = (
+                "Rule would have been added to "
+                "mysql_query_rules_fast_routing, "
+                "however check_mode is enabled."
+            )
+
+    def update_rule(self, check_mode, result, cursor):
+        if not check_mode:
+            result['changed'] = self.update_rule_config(cursor)
+            result['msg'] = "Updated rule in mysql_query_rules_fast_routing"
+            self.manage_config(cursor, result['changed'])
+            result['rules'] = self.get_rule_config(cursor)
+        else:
+            result['changed'] = True
+            result['msg'] = (
+                "Rule would have been updated in "
+                "mysql_query_rules_fast_routing, "
+                "however check_mode is enabled."
+            )
+
+    def delete_rule(self, check_mode, result, cursor):
+        if not check_mode:
+            result['rules'] = self.get_rule_config(cursor)
+            result['changed'], result['rows_affected'] = self.delete_rule_config(cursor)
+            result['msg'] = "Deleted rule from mysql_query_rules_fast_routing"
```

Comment:
```suggestion
            result['msg'] = "Deleted rule from mysql_query_rules_fast_routing."
```
As it starts with a capital letter.